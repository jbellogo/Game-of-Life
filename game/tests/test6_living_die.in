new 9
init
1 1
4 1
7 1

1 4
4 4
7 4

1 7
4 7
7 7
-1 -1
print

init 
4 0

7 0
8 1

1 3
0 3
0 4

3 3
4 3
4 5
3 4


7 3
7 5
6 4
6 5
8 3
8 4
8 5

1 6
1 8
0 6
0 7
0 8
2 6
2 7
2 8


-1 -1
print

step 
print 
step
print 
step 
print
steps 12
print
step
print
steps 20
print

