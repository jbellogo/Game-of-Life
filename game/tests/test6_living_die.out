_________
_X__X__X_
_________
_________
_X__X__X_
_________
_________
_X__X__X_
_________
____X__X_
_X__X__XX
_________
XX_XX__XX
XX_XX_XXX
____X_XXX
XXX______
XXX_X__X_
XXX______
_______XX
_______XX
XXXXX____
XX_XXXX_X
XX_______
____X_X_X
X_X__XX_X
_________
X_XX_____
_______XX
_XXX___XX
X_____X_X
_____X___
XXXX__X__
X_____X__
_____XX__
__XX_____
_________
__X____XX
_XX___X__
_XX___X_X
X_X__XXX_
XXX__XX__
X_X___XX_
_____XX__
_________
_________
_________
_________
_________
_________
__X______
__X___XX_
__X__X__X
______XX_
_________
_________
_________
_________
_________
_________
_XXX__XX_
_____X__X
______XX_
_________
_________
_________
_________
_________
_________
_XXX__XX_
_____X__X
______XX_
_________
